lms/envs/common.py

```diff
@@ -773,7 +773,7 @@
 CMS_BASE = 'localhost:8001'
 
 # Site info
-SITE_ID = 1
+# SITE_ID = 1
```

Comment:
If this setting is no longer relevant we should just remove it vs. hiding it.  If it is still useful but we want to hide it we should add a comment explaining the scenario(s) in which it should be enabled.